src/card.tsx

```diff
@@ -281,10 +270,32 @@ const useVideoWithControls = (
         }
     };
 
+    const videoAction: CardAction = {
+        Icon: {
+            playing: IconPauseFilled,
+            loading: IconPauseFilled,
+            paused: IconPlayFilled,
+            loadingTimeout: VideoSpinner,
+            error: undefined,
+        }[videoStatus],
+        onPress: onVideoControlPress,
+        label: {
+            playing: texts.pauseIconButtonLabel,
+            loading: texts.pauseIconButtonLabel,
+            paused: texts.playIconButtonLabel,
+            loadingTimeout: '',
+            error: '',
+        }[videoStatus],
+        disabled: !['loading', 'playing', 'paused'].includes(videoStatus),
```

Comment:
When the action is disabled, the hover/active states are still there. I don't think that's expected

Reply:
fixed